py/examples/picker.py

```diff
@@ -14,7 +14,7 @@ async def serve(q: Q):
         ]
     else:
         q.page['example'] = ui.form_card(box='1 1 4 10', items=[
-            ui.picker(name='picker', label='Place an order (try Spam, Eggs or Ham):', choices=[
+            ui.picker(name='picker', trigger=True, label='Place an order (try Spam, Eggs or Ham):', choices=[
```

Comment:
This example is unrelated to the combobox. Is this an intentional change?

Reply:
I was checking the picker behavior on trigger and commited it by mistake. Reverted, thanks.